contrib/lib/src/serve.rs

```diff
@@ -314,6 +315,15 @@ impl StaticFiles {
         self.rank = rank;
         self
     }
+
+    /// Sets the default content type
+    ///
+    /// If a file does not have an extension, this value will be used
+    /// for the content-type header
+    pub fn default_type(mut self, content_type: ContentType) -> Self {
```

Comment:
```suggestion
    pub fn with_default(mut self, content_type: ContentType) -> Self {
```

To match `with_default` in `NamedFile`.